docs/js-core-1/week-1/lesson.md

```diff
@@ -251,80 +345,143 @@ function double(number) {
 }
 ```
 
-To use the function we need to:
-a) _call_ it with an input and
-b) do something with the returned value, for example assigning it to a variable
+To use a function then we we need to **call** it ( some people will say **invoke** )
 
 ```js
 const result = double(2);
 
 console.log(result); // 4
 ```
 
-The input given to a function is called a **parameter**.
+The function `double` is being invoked by placing parathenses after the name of the function like this `double()`
```

Comment:
Given that we say "call" as the main verb above, and most volunteers in my experience use that, could we change to:

```suggestion
The function `double` is being called by placing parathenses after the name of the function like this `double()`
```